config.py

```diff
@@ -1,10 +1,86 @@
-import math as maths
-
-Width = 50  # C 50 How many squares wide the board is
-Height = 30  # C 30 Ditto but with height
-Size = 20  # C 20 The size of the sides of each square (in pixels)
-Cushion = 5  # C 10 How far the board extends beyond the visible amount
-Edge = maths.ceil(Size / 7)  # C Size/15 The gap between each cell
-Background = (120, 120, 120)  # C (120,120,120) The colour of the background
+import pygame
+
 Dead = 0
 Square = 1
```

Comment:
I've used [enums](https://docs.python.org/3/library/enum.html) in my pool game for striped balls and current player values. You can check it out, It does make the code more readable, but it's not an easy task to implement.[ Here's an example from my code](https://github.com/max-kov/pool/blob/f0a03fb9e928f7cfa247ba5e9a8c3944148b1a31/pool/gamestate.py#L201)